### PR TITLE
add Indonesian translation

### PR DESCRIPTION
**Pull request by bmx666**
Hi @Gamefire238
Please review the translation made by ChatGPT-4, if you find something, please leave a comment in the appropriate line about the correct translation.

### CODE REVIEW

File: app/src/main/res/values-in/strings.xml [hunk 1]
```diff
@@ -1,5 +1,363 @@
...
+    <string name="service_name">Pembersih Cache</string>
+    <string name="tile_name">Pembersih Cache</string>
+    <string name="text_enable_accessibility_title">Aktifkan Layanan Aksesibilitas</string>
+    <string name="text_enable_accessibility_message">
+        Untuk berfungsi dengan baik, <b>Pembersih Cache</b> memerlukan izin Anda.
+        Pergi ke Pengaturan untuk memberikan izin untuk <b>Aksesibilitas</b>.
+        \n\n
+        1. Pergi ke <b>Aksesibilitas</b> di Pengaturan.
+        \n
+        2. Temukan <b>Pembersih Cache</b> dan ketuk pada itu.
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:13]: 2. Temukan <b>Pembersih Cache</b> dan ketuk disana.

File: app/src/main/res/values-in/strings.xml [hunk 2]
```diff
@@ -1,5 +1,363 @@
...
+        \n
+        - Aplikasi ini tidak mengumpulkan dan/atau membagikan data pribadi atau sensitif menggunakan kemampuan Aksesibilitas.
+    </string>
+    <string name="text_enable_accessibility_message_api33">
+        Untuk berfungsi dengan baik, <b>Pembersih Cache</b> memerlukan izin Anda.
+        Pergi ke Pengaturan untuk memberikan izin untuk <b>Aksesibilitas</b>.
+        \n\n
+        1. Pergi ke <b>Aksesibilitas</b> di Pengaturan.
+        \n
+        2. Temukan <b>Pembersih Cache</b> dan ketuk pada itu.
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:31]: 2. Temukan <b>Pembersih Cache</b> dan ketuk disana.

File: app/src/main/res/values-in/strings.xml [hunk 3]
```diff
@@ -1,5 +1,363 @@
...
+        "Pembersih Cache" memerlukan API Layanan Aksesibilitas untuk menerapkan fungsi intinya, seperti melakukan klik, gesekan dan fungsi inti lainnya.
+        "Pembersih Cache" tidak mengumpulkan dan/atau membagikan data pribadi atau sensitif menggunakan kemampuan Aksesibilitas.
+    </string>
+    <string name="text_enable_usage_stats_title">Aktifkan Statistik Penggunaan</string>
+    <string name="text_enable_usage_stats_message">
+        Untuk berfungsi dengan baik, <b>Pembersih Cache</b> memerlukan izin Anda. Pergi ke Pengaturan untuk memberikan izin untuk Statistik Penggunaan.
+        \n\n
+        1. Pergi ke <b>Aplikasi</b> di Pengaturan.
+        \n
+        2. Temukan <b>Akses aplikasi khusus</b> dan ketuk pada itu.
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:58]: 2. Temukan <b>Akses aplikasi khusus</b> dan ketuk disana.

**Comment by bmx666** [comment]
Thanks @Gamefire238,

I replaced everywhere `dan ketuk pada itu` on `dan ketuk disana`

Do you want to suggest anything else or is everything ok?

File: app/src/main/res/values-in/strings.xml [hunk 4]
```diff
@@ -1,5 +1,363 @@
...
+    <string name="btn_open_accessibility">Buka Pengaturan Aksesibilitas</string>
+    <string name="btn_open_usage_stats">Buka Pengaturan Statistik Penggunaan</string>
+    <string name="btn_clean_all_cache">Bersihkan cache semua aplikasi</string>
+    <string name="btn_clean_system_cache">Bersihkan cache aplikasi sistem</string>
+    <string name="btn_clean_user_cache">Bersihkan cache aplikasi pengguna</string>
+    <string name="btn_close_app">Tutup aplikasi</string>
+    <string name="btn_start_accessibility_service">Mulai layanan Aksesibilitas</string>
+    <string name="btn_stop_accessibility_service">Berhenti layanan Aksesibilitas</string>
+    <string name="btn_add_extra_search_text">Tambahkan teks pencarian tambahan</string>
+    <string name="description_apps_all_check">Cek semua aplikasi</string>
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:90]: <string name="description_apps_all_check">Centang semua aplikasi</string>

**Comment by Gamefire238** [comment]
Actually it's still there, it's just that doing it in Chrome android is terrible.

Maybe I will continue it another time, thanks

File: app/src/main/res/values-in/strings.xml [hunk 5]
```diff
@@ -1,5 +1,363 @@
...
+    <string name="description_apps_all_uncheck">Uncek semua aplikasi</string>
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:91]: <string translatable="false" name="description_apps_all_uncheck">Hapus centang semua aplikasi</string>

File: app/src/main/res/values-in/strings.xml [hunk 6]
```diff
@@ -1,5 +1,363 @@
...
+        <br/>
+        2. Aktifkan dukungan Statistik Penggunaan untuk aplikasi <b>Pembersih Cache</b> jika diperlukan
+        <br/>
+        3. Aktifkan dukungan Aksesibilitas untuk aplikasi <b>Pembersih Cache</b> jika diperlukan
+        <br/>
+        4. Periksa aplikasi yang diperlukan dalam daftar (gunakan tombol mengambang untuk mencentang/menghilangkan centang semua aplikasi)
+        <br/>
+        5. Tekan <b>tombol mengambang Bersihkan Cache</b> untuk menjalankan proses pembersihan cache
+        <br/>
+        6. Tekan <b>tombol mengambang Aksesibilitas</b> untuk mengganggu proses ATAU tunggu hingga proses selesai
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:219]: 6. Tekan <b>tombol mengambang Aksesibilitas</b> untuk menghentikan proses ATAU tunggu hingga proses selesai

File: app/src/main/res/values-in/strings.xml [hunk 7]
```diff
@@ -1,5 +1,363 @@
...
+        <br/>
+        7. <i>(opsional)</i> Tekan <b>Berhenti layanan Aksesibilitas</b> untuk menonaktifkan layanan Aksesibilitas
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:221]: 7. <i>(opsional)</i> Tekan <b>Hentikan layanan Aksesibilitas</b> untuk menonaktifkan layanan Aksesibilitas

File: app/src/main/res/values-in/strings.xml [hunk 8]
```diff
@@ -1,5 +1,363 @@
...
+    <string name="text_clean_cache_interrupt_display_size">Pembersihan cache terganggu\nMembersihkan %1$s</string>
+    <string name="text_clean_cache_interrupt_processing">Pembersihan cache terganggu\nSedang diproses…</string>
+    <string name="btn_open_accessibility">Buka Pengaturan Aksesibilitas</string>
+    <string name="btn_open_usage_stats">Buka Pengaturan Statistik Penggunaan</string>
+    <string name="btn_clean_all_cache">Bersihkan cache semua aplikasi</string>
+    <string name="btn_clean_system_cache">Bersihkan cache aplikasi sistem</string>
+    <string name="btn_clean_user_cache">Bersihkan cache aplikasi pengguna</string>
+    <string name="btn_close_app">Tutup aplikasi</string>
+    <string name="btn_start_accessibility_service">Mulai layanan Aksesibilitas</string>
+    <string name="btn_stop_accessibility_service">Berhenti layanan Aksesibilitas</string>
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:88]: Hentikan layanan Aksesibilitas

File: app/src/main/res/values-in/strings.xml [hunk 9]
```diff
@@ -1,5 +1,363 @@
...
+        <br/>
+        2. Klik <b>Skenario Bersihkan cache</b> dan pilih <b>Xiaomi MIUI</b>
+        <br/>
+        <br/>
+        <h4>PERHATIAN ⚠️: Smartphone Huawei dan Xiaomi sering menggunakan optimalisasi baterai yang agresif yang
+        dapat mengganggu komunikasi antara <b>Pembersih Cache</b> dan layanan Aksesibilitas,
+        mencegah mereka bekerja sesuai harapan. Silakan ikuti langkah-langkah di bawah ini:</h4>
+        1. Buka menu <b>Pengaturan</b> di pojok kanan atas dan temukan bagian <b>Tampilkan tombol tambahan</b>
+        <br/>
+        2. Aktifkan opsi <b>Mulai/Berhenti layanan Aksesibilitas</b>
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:271]: Hentikan layanan Aksesibilitas

File: app/src/main/res/values-in/strings.xml [hunk 10]
```diff
@@ -1,5 +1,363 @@
...
+        <br/>
+        3. Kali berikutnya Anda memulai <b>Pembersih Cache</b>, jika layanan Aksesibilitas berjalan,
+        hentikan dengan mengklik tombol <b>Berhenti layanan Aksesibilitas</b>.
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:274]: Hentikan layanan Aksesibilitas

File: app/src/main/res/values-in/strings.xml [hunk 11]
```diff
@@ -1,5 +1,363 @@
...
+    <string name="prefs_title_filter_min_cache_size">Ukuran Cache Min. untuk menyaring aplikasi</string>
+    <string name="prefs_summary_filter_min_cache_size">Min. %1$s</string>
+    <string name="prefs_error_convert_filter_min_cache_size">Silakan masukkan ukuran cache min. dalam KB,MB,GB,TB ATAU 0 untuk mereset penyaring</string>
+    <string name="prefs_title_filter_hide_disabled_apps">Sembunyikan aplikasi yang dinonaktifkan</string>
+    <string name="prefs_title_filter_hide_ignored_apps">Sembunyikan aplikasi yang diabaikan</string>
+    <string name="prefs_title_filter_show_dialog_to_ignore_app">Tampilkan dialog untuk mengabaikan aplikasi</string>
+    <string name="prefs_title_filter_list_of_ignored_apps_edit">Daftar aplikasi yang diabaikan</string>
+    <string name="prefs_summary_filter_list_of_ignored_apps_edit">Edit daftar aplikasi yang diabaikan</string>
+    <string name="prefs_show_extra_buttons">Tampilkan tombol tambahan</string>
+    <string name="prefs_title_show_button_start_stop_service">Mulai/Berhenti Layanan Aksesibilitas</string>
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:328]: Hentikan layanan Aksesibilitas

**Comment by bmx666** [comment]
no problem, take your time! Feel free to review later.

File: app/src/main/res/values-in/strings.xml [hunk 12]
```diff
@@ -1,5 +1,363 @@
...
+    <string name="prefs_summary_show_button_start_stop_service">Untuk beberapa smartphone, Anda perlu secara manual memulai atau menghentikan layanan aksesibilitas</string>
+    <string name="prefs_title_show_button_close_app">Tutup aplikasi</string>
+    <string name="prefs_summary_show_button_close_app">Untuk beberapa smartphone, Anda perlu secara manual menghentikan layanan aksesibilitas dan menutup aplikasi</string>
+    <string name="prefs_extra_actions">Tindakan tambahan</string>
+    <string name="prefs_title_extra_action_stop_service">AutoStop Layanan Aksesibilitas</string>
+    <string name="prefs_summary_extra_action_stop_service">Secara otomatis menghentikan Layanan Aksesibilitas setelah membersihkan cache</string>
+    <string name="prefs_title_extra_action_close_app">AutoClose aplikasi</string>
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:335]: Tutup otomatis aplikasi

File: app/src/main/res/values-in/strings.xml [hunk 13]
```diff
@@ -1,5 +1,363 @@
...
+    <string name="prefs_title_filter_show_dialog_to_ignore_app">Tampilkan dialog untuk mengabaikan aplikasi</string>
+    <string name="prefs_title_filter_list_of_ignored_apps_edit">Daftar aplikasi yang diabaikan</string>
+    <string name="prefs_summary_filter_list_of_ignored_apps_edit">Edit daftar aplikasi yang diabaikan</string>
+    <string name="prefs_show_extra_buttons">Tampilkan tombol tambahan</string>
+    <string name="prefs_title_show_button_start_stop_service">Mulai/Berhenti Layanan Aksesibilitas</string>
+    <string name="prefs_summary_show_button_start_stop_service">Untuk beberapa smartphone, Anda perlu secara manual memulai atau menghentikan layanan aksesibilitas</string>
+    <string name="prefs_title_show_button_close_app">Tutup aplikasi</string>
+    <string name="prefs_summary_show_button_close_app">Untuk beberapa smartphone, Anda perlu secara manual menghentikan layanan aksesibilitas dan menutup aplikasi</string>
+    <string name="prefs_extra_actions">Tindakan tambahan</string>
+    <string name="prefs_title_extra_action_stop_service">AutoStop Layanan Aksesibilitas</string>
```
Comment (Gamefire238) [inline app/src/main/res/values-in/strings.xml:333]: Hentikan otomatis Layanan Aksesibilitas

**Review by Gamefire238 — COMMENTED** [review]
Finally finished, thank you friends for the convenience

**Comment by bmx666** [comment]
Thanks a lot @Gamefire238 for your contribution!

**Pull request merged**